src/record/list.js

```diff
@@ -214,7 +226,9 @@ List.prototype._applyUpdate = function( message ) {
 		message.data[ 2 ] = '[]';
```

Comment:
i know this isn't related, but this means you could listen to any record as a list and be told its empty rather than it not being a list.